specification/synapse/data-plane/Microsoft.Synapse/preview/2022-03-01-preview/runNotebook.json

```diff
@@ -0,0 +1,562 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-03-01-preview",
+    "title": "ArtifactsClient"
```

Comment:
Is this really the "ArtifactsCient"?  There is already another API definition with this title in this folder.  Is this a copy/paste error?

Reply:
Yes, it should be "ArtifactsCient". We will put these Swagger specs with the same title together. If we change it to something else, code generation will fail. 

Error message:
```
error : The 'title' across provided OpenAPI definitions has to match. Found: 'RunNotebookClient', 'ArtifactsClient'.
```